### PR TITLE
:sparkles: Add new exercise : Affine Cipher

### LINKED ISSUE #886: Add new exercise : Affine cipher
[problem-specifications](https://github.com/exercism/problem-specifications/blob/master/exercises/affine-cipher/description.md) | [metadata](https://github.com/exercism/problem-specifications/blob/master/exercises/affine-cipher/metadata.yml) | [canonical-data](https://github.com/exercism/problem-specifications/blob/master/exercises/affine-cipher/canonical-data.json)

### PR DESCRIPTION
**Pull request by TomPradat**
Fix #886

### CODE REVIEW

File: exercises/affine-cipher/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "exercism-javascript",
+  "description": "Exercism exercises in Javascript.",
+  "author": "Tom Pradat",
```
Comment (TomPradat) [inline exercises/affine-cipher/package.json:4]: Am I supposed to put my name here ?

File: exercises/affine-cipher/example.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+const ALPHABET = 'abcdefghijklmnopqrstuvwxyz';
```
Comment (TomPradat) [inline exercises/affine-cipher/example.js:1]: This file is still in WIP. I just wrote a quick and dirty solution

File: exercises/affine-cipher/package.json (hunk 1 above)
Comment (TomPradat) [reply exercises/affine-cipher/package.json]: I suppose not, the CI fails if author is not "Katrina Owen"

Comment (SleeplessByte) [reply exercises/affine-cipher/package.json]: We will use the v3 format to get your authorship in!

File: config.json [hunk 3]
```diff
@@ -1464,6 +1464,19 @@
       "topics": [
         "graph_theory"
       ]
+    },
```
Comment (SleeplessByte) [inline config.json:1467]: I think this file is _ordered_. Or at least it was. It should be around the other `unlocked_by` `simple-cipher`, and then it's probably sorted by difficulty > lexographically.

File: exercises/affine-cipher/affine-cipher.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export const encode = (phrase, key) => {
+  throw new Error("Remove this statement and implement this function");
+};
+
+export const decode = (phrase, key) => {
+  throw new Error("Remove this statement and implement this function");
+}
```
Comment (SleeplessByte) [inline exercises/affine-cipher/affine-cipher.js:7]:
```suggestion
};

```

File: exercises/affine-cipher/example.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+const m = ALPHABET.length;
```
Comment (SleeplessByte) [inline exercises/affine-cipher/example.js:2]: Can you uppercase this, and give it a name?

**Review by SleeplessByte — APPROVED** [review]
2 nits :). Looking great @TomPradat

**Comment by SleeplessByte** [comment]
You're on a roll! Are you on our maintainers list yet? Would you like to help out with v3?

**Comment by TomPradat** [comment]
> You're on a roll! Are you on our maintainers list yet? Would you like to help out with v3?

Idk if I'm on the maintainers list yet, i guess not. I've already started a PR to help with v3 and I'll be glad to help more 👍

**Pull request merged**